### DISCUSSION TITLE
Enable and disable turbo mode from a batch file or command?

### DISCUSSION THREAD

**Discussion by fcoury** [joncampbell123/dosbox-x#5198] [General]
I am running dosbox-x to run a CA-Clipper compiler and the linker is really slow. I don't always remember to click the Turbo option from the menu so I was wondering if there's a way to toggle the turbo mode from within the guest machine?

Thanks!

**Comment by maron2000** [comment]
You can map a key to `Turbo (Fast Forward)` in the mapper editor (See 10/10, make sure that the mapped key doesn't interfere the software you're running).
If you run the compiler in a batch file insert, `config -set turbo=true`.

**Comment by fcoury** [comment]
That's excellent to know, thank you!
